src/Client/Client.php

```diff
@@ -75,6 +77,7 @@ public function setPropertyAccessor(PropertyAccess\PropertyAccessorInterface $pr
      * @param string         $name  the event name
      *
      * @throws Exception
+     * @return void
```

Comment:
We have to do something about this :cry: 
https://github.com/M6Web/Symfony2-coding-standard/issues/14